packages/plugin-doc-coverage/mocks/component-mock.unit.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, expect, it } from 'vitest';
+import { DUMMY_FUNCTION, DUMMY_FUNCTION_2 } from './component-mock.js';
+
+export function shouldnotBeHere() {
```

Comment:
See https://github.com/code-pushup/cli/pull/896/files#r1894665959
lets reorganise the mocks